Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Queries/ListIdentities/HandlerTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Backbone.Modules.Devices.Infrastructure.Persistence.Database;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using FakeItEasy;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Xunit;
+using Backbone.Modules.Devices.Application.Tests;
+using Enmeshed.UnitTestTools.TestDoubles.Fakes;
+using Backbone.Modules.Devices.Domain.Entities;
+using Backbone.Modules.Devices.Application.Identities.Queries.ListIdentities;
+using Handler = Backbone.Modules.Devices.Application.Identities.Queries.ListIdentities.Handler;
+using Enmeshed.BuildingBlocks.Application.Pagination;
+using Backbone.Modules.Devices.Application.AutoMapper;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Identities.Queries.ListIdentities;
+
+public class HandlerTests
+{
+    private static readonly IdentityAddress ActiveIdentity = TestDataGenerator.CreateRandomIdentityAddress();
+
+    private readonly DevicesDbContext _arrangeContext;
+    private readonly DevicesDbContext _actContext;
+    private readonly Handler _handler;
+
+    public HandlerTests()
+    {
+        AssertionScope.Current.FormattingOptions.MaxLines = 1000;
+
+        (_arrangeContext, _, _actContext) = FakeDbContextFactory.CreateDbContexts<DevicesDbContext>();
+        _handler = CreateHandler();
+    }
+
+    [Fact]
+    public async void Returns_empty_list_when_no_loaded_entities()
+    {
+        // Arrange
+        List<Identity> identitiesList = new();
+        _arrangeContext.SaveEntities(identitiesList.ToArray());
+        var request = new PaginationFilter() { PageSize = 5 };
+
+        // Act
+        var result = await _handler.Handle(new ListIdentitiesQuery(request), CancellationToken.None);
+
+        // Assert
+        result.Identities.Should().HaveCount(0);
+    }
+
+    [Fact]
+    public async void Returns_valid_list_when_entities_loaded()
```

Comment:
What is a "**valid** list"?

Reply:
I suppose by valid list I meant simply "a list"